website/docs/docs/building-a-dbt-project/building-models/snowflake-configs.md

```diff
@@ -139,6 +139,26 @@ create or replace table my_database.my_schema.my_table as (
  alter table my_database.my_schema.my_table cluster by (session_start);"
 ```
 
+It is possible to use functions of fields to cluster by, as below:
```

Comment:
I think saying "expressions" might be clearer than "functions of fields" here - you buy that?

```suggestion
It is possible to cluster by expressions, as below:
```